installers/nersc-dev/radiasoft-download.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# To run: curl radia.run | bash -s nersc-dev
+#
+nersc_dev_main() {
+    local r=~/.pyenv
+    if [[ ! -d $r ]]; then
+        git clone https://github.com/pyenv/pyenv.git "$r"
```

Comment:
why not use `curl https://pyenv.run | bash`?

Reply:
8ffe45a